### DISCUSSION TITLE
这个有考虑可视化操作吗

### DISCUSSION THREAD

**Discussion by JCYTop** [openupm/openupm#4902] [General]
这个有考虑可视化操作吗

**Comment by favoyang** [comment]
No plans for that.

Unity restricts third-party package managers within the Editor to maintain a certain level of control over the ecosystem. It's not something the project can bypass.

**Comment by AlianBlank** [comment]
I have incorporated visual operations into my framework. The following is a preview image. And the warehouse address

https://github.com/GameFrameX/com.gameframex.unity
![image](https://github.com/user-attachments/assets/a29575eb-0634-49de-a2a3-c992a6b133d7)
